src/main/java/seedu/address/model/AddressBook.java

```diff
@@ -56,57 +59,153 @@ public void setPersons(List<Person> persons) {
         this.persons.setPersons(persons);
     }
 
+    /**
+     * Replaces the contents of the student list with {@code students}.
+     * {@code students} must not contain duplicate students.
+     */
+    public void setStudents(List<Person> students) {
```

Comment:
Perhaps we should use wildcard here? i.e. List<? extends Person>

Reply:
ok